cli/isolate_state.rs

```diff
@@ -42,9 +49,16 @@ pub struct IsolateState {
   pub modules: Mutex<Modules>,
   pub worker_channels: Option<Mutex<WorkerChannels>>,
   pub global_timer: Mutex<GlobalTimer>,
+  pub workers: Mutex<WebWorkerTable>,
```

Comment:
Does this include the main isolate?

Reply:
Part of the main isolate state? yes. Main isolate stored here? no. This is so we can have some enforcement of ownership on workers, and provide a shared future for when `op_host_get_worker_closed`(where polling the worker isolate is scheduled) is called.